src/conversion/view-v1-to-v2.js

```diff
@@ -71,18 +71,15 @@ function convertRenderer (cell) {
   if (renderer === undefined) {
     return
   }
-  switch (renderer) {
-    case 'boolean':
-    case 'string':
-    case 'number':
-      return {name: renderer}
-    case 'button-group':
-    case 'select':
-    case 'multi-select':
-    case 'textarea':
-    default:
-      return customRenderer(renderer, cell.properties)
+  const basicRenderers = [
```

Comment:
The lint rules were complaining about the complexity of this function, so I refactored it to check for the first 3 cases and do their renderers, otherwise we treat it as a custom renderer. It should behave functionally the same as the switch statement since it had a default case.